deployments/aws.yml

```diff
@@ -15,6 +15,7 @@
 #    Replace the value of the `ECS_CLUSTER` environment variable in the workflow below with the name you set for the cluster.
 #
 # 3. Store your ECS task definition as a JSON file in your repository.
+#    Get task definiton. For example, `aws ecs describe-task-definition --task-definition {taskdefinition} --output json`
```

Comment:
I don't have much knowledge on this, but we are already suggesting a different command in next line `aws ecs register-task-definition --generate-cli-skeleton`.